src/js/routes/nodes.js

```diff
@@ -45,12 +46,8 @@ let nodesRoutes = {
         {
           type: Route,
           name: 'nodes-task-panel',
-          path: 'task-detail/:taskID/?'
-        },
-        {
-          type: Route,
-          name: 'node-detail-tasks',
-          path: 'task-detail/:taskID/?'
+          path: 'task-detail/:taskID/?',
+          handler: TaskDetail
```

Comment:
We should probably wrap this in a NodeDetailPage handler though, so that we can display the Nodes breadcrumbs. Without that, when you click on a Task from the Nodes page you end up in a navigation dead end.